adserver/models.py

```diff
@@ -974,6 +1002,31 @@ def show_to_geo(self, geo_data):
             ):
                 return False
 
+        # Check if the country traffic cap exceeds the current fill for that country
+        if self.traffic_cap and "countries" in self.traffic_cap:
+            # pylint: disable=invalid-sequence-index
```

Comment:
Weird it's raising this issue when traffic_cap is a dict. Is it because of `traffic_fill.get("countries", {})` or something? 

Seems like it shouldn't apply here: 

https://pylint.readthedocs.io/en/latest/user_guide/messages/error/invalid-sequence-index.html

Reply:
Ya, it definitely triggers a lint issue and I'm not sure why. It shouldn't.